src/app/components/channel/channel.component.html

```diff
@@ -1,5 +1,14 @@
-<div class="card" fxLayout="column" fxLayoutAlign="space-evenly">
-    <div fxLayout="row" fxLayoutAlign="start center">
+<div
+    class="card"
+    [ngClass]="
+        channel.state !== 'opened' && channel.state !== 'waiting_for_open'
+            ? 'card--transparent'
+            : ''
+    "
```

Comment:
Nitpick: iirc, this could be made clearer with the object syntax of `ngClass`, like
```suggestion
    [ngClass]="{ 'card--transparent': channel.state !== 'opened' && channel.state !== 'waiting_for_open' }"
```

Reply:
I forgot about the object syntax. Thank you!